src/core/Akka.Persistence.TCK/Akka.Persistence.TCK.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\contrib\persistence\Akka.Persistence.Query.Sql\Akka.Persistence.Query.Sql.csproj" />
```

Comment:
I need to access some of its internal class in the test so I can write a test to check that the buffer is actually released when it ran out of subscribers.

Reply:
The TCK should work broadly, without any connection to a specific plugin, since the TCK is what's used to validate all plugins. We'll need to clean that up.

Reply:
Got it, I'll move it to Query.Sql.Tests instead.

Reply:
Done, moved to PersistenceIdsSpec instead